src/components/Card/Card.jsx

```diff
@@ -51,7 +51,11 @@ const Card = ({ cardArray }) => {
             buttonText="Details"
             onClick={() => setActiveDetails(card)}
           />
-          <Button buttonName="remove" buttonText="Remove" />
+          <Button
+            buttonName="remove"
+            buttonText="Remove"
+            onClick={() => onClickPrimary(card._id)}
+          />
```

Comment:
Let's make this generic, the card shouldn't always have a remove button. What about the home page? That has a "save" button

Reply:
done